src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -2022,6 +2026,14 @@ DiagnosticBag getDiagnosticBag()
             bool takeSnapshots,
             out NullableWalker.SnapshotManager snapshotManager,
             ref ImmutableDictionary<Symbol, Symbol>? remappedSymbols);
+
+#if DEBUG
+        /// <summary>
+        /// Performs just the analysis step of getting nullability information for a semantic model. This is only used during DEBUG builds where nullable
+        /// is turned off on a compilation level, giving some extra debug verification of the nullable flow analysis. 
+        /// </summary>
+        protected abstract void AnalyzeBoundNodeNullability(BoundNode boundRoot, Binder binder, DiagnosticBag diagnostics, bool takeSnapshots);
```

Comment:
The declaration of this method has the parameter name `takeSnapshots` but all of the `overrides` have the parameter `craeteSnapshots`. Feel like we should be consistent here.